packages/client/src/modules/contact/containers/Contact.jsx

```diff
@@ -1,38 +1,34 @@
 import React from 'react';
-import { graphql, compose } from 'react-apollo';
+import { Mutation } from 'react-apollo';
 
 import ContactView from '../components/ContactView';
 
 import CONTACT from '../graphql/Contact.graphql';
 
 class Contact extends React.Component {
```

Comment:
There is no longer need to declare Contact element as a class, declare it as a closure-function instead.